docs/content/en/use-cases/spread-select.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: Spread `SELECT`s from reporting apps among cluster nodes
+title: Spread SELECTs from reporting apps among cluster nodes
```

Comment:
Could keep it if was reflected on the website but the backticks propage to `og:title`meta.